util/buildRelease/install.sh

```diff
@@ -233,9 +239,13 @@ myinstallfileto () {
   fi
 }
 
-# run 'cmake' to install the compiler library, 'chpl' and 'chpldoc'
+# these makefile targets run 'cmake' to install the compiler library, 'chpl',
+# and optionally 'chpldoc'
+# if [ $BUILD_CHPLDOC -eq 1 ]
```

Comment:
why is this commented out?

Reply:
also should have gone with the other commented code